remme/rpc_api/utils.py

```diff
@@ -34,7 +34,10 @@ def _get_first_error(message):
         @functools.wraps(func)
         async def wrapper(request, *args, **kwargs):
             logger.debug(f'Req params: {request.params}')
-            form = form_class(ignore_fields=ignore_fields, **request.params)
+            try:
+                form = form_class(ignore_fields=ignore_fields, **request.params)
+            except Exception as e:
+                raise RpcInvalidParamsError(message=str(e))
```

Comment:
Why this has been added?

Reply:
Because constructor of ListBatchesForm can throw exceptions.

Reply:
@Art17, could we move this to another place? For instance, to view code or form code. I am afraid of this abstraction (validate params code) too unreliable to modify or extend it.

Reply:
Done.